src/js/components/List/List.js

```diff
@@ -56,6 +48,8 @@ const List = React.forwardRef(
   ) => {
     const theme = useContext(ThemeContext);
     const [active, setActive] = useState();
+    const [focus, setFocus] = useState();
```

Comment:
We're trying to move away from this pattern of tracking focus in javascript and instead relying on having `&:focus` in the CSS.

Reply:
Thanks for the reminder, I've updated it back to use CSS.